src/main/java/org/opentripplanner/netex/mapping/QuayMapper.java

```diff
@@ -59,6 +60,21 @@ RegularStop mapQuayToStop(
 
     builder.fareZones().addAll(fareZones);
 
+    /* handle IFOTP id stored in KV GlobalID in the DELFI feed to allow matching to OSM */
+    if (quay.getKeyList() != null) {
+      String globalId = quay
+        .getKeyList()
+        .getKeyValue()
+        .stream()
+        .filter(keyValueStructure -> keyValueStructure.getKey().equals("GlobalID"))
```

Comment:
I think this is ok, but we should make the key configurable in the NeTEx configuration

Reply:
I agree. I have looked at the existing NeTEx test data and noticed ids in other keys. It also nicely matches to the OSM side of the configuration.

Reply:
I started the implementation with one global configration option. (basically copying from the OSM option)
But I'm wondering if it's better to make it an option of the NeTEx source.
So a use case like netherlands+belgium+germany (as someone else asked about) can use three different tags. (as only a single tag can be easily added)

Reply:
The configuration should be in the default + per source, similar to how feedId is https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/src/main/java/org/opentripplanner/standalone/config/buildconfig/NetexConfig.java#L31-L49

Reply:
I have adjusted the epip smoke test to use the default NeTEx configuration, the nordic smoke test uses the empty default, source specific configuration was locally tested.

Missing is a test that setting both default and specific configuration uses the right value and a test with multiple feeds with different values.

I did not see an elegant way to DRY by adding a "getKeyValue" to some "extendable NeTEx element type" base class.

Reply:
I think the added configuration parameter needs to https://github.com/opentripplanner/OpenTripPlanner/labels/bump%20serialization%20id

Reply:
I dropped the unrelated changes, rebased to dev-2.x and squashed it. Should be ready for merge now